### PR TITLE
Add support for building against Postgres 13

### PR DESCRIPTION
**Pull request by lfittl**
Depends on https://github.com/citusdata/tools/pull/78 for Postgres 13 testing on Travis.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -17,6 +17,7 @@ matrix:
     - env: PGVERSION=10
     - env: PGVERSION=11
     - env: PGVERSION=12
+    - env: PGVERSION=13
     - env: LINTING=true
 before_install:
   - git clone -b update-uncrustify-on-travis --depth 1 https://github.com/citusdata/tools.git
```
Comment (mtuncer) [inline .travis.yml:23]: we usually use a tag instead of branch

**Comment by hanefi** [comment]
I triggered the tests using the updated scripts in https://github.com/citusdata/tools/pull/78

https://travis-ci.com/github/citusdata/pg_auto_failover/builds/172101872

The tests seem to run fine. However not all the tests passed for PG13, and also we have some linter errors.

File: .travis.yml (hunk 1 above)
Comment (hanefi) [reply .travis.yml:23]: I think we should create a new tag on tools develop branch after https://github.com/citusdata/tools/pull/78 gets merged. Then use the new tag name in here.

Possibly `0.7.14` or `0.8.0` maybe? I don't really understand the versioning schema on that repository.

**Comment by lfittl-msft** [comment]
@hanefi @DimCitus @mtuncer FYI, re-opening this as #312 with my MSFT alias (and a branch directly on this repo)

**Pull request closed without merging**